lib/helper.c

```diff
@@ -326,8 +326,13 @@ int fuse_main_real(int argc, char *argv[], const struct fuse_operations *op,
 
 	if (opts.singlethread)
 		res = fuse_loop(fuse);
-	else
-		res = fuse_loop_mt(fuse, opts.clone_fd);
+	else {
+		struct fuse_loop_config loop_config;
+		struct fuse_config *config = fuse_get_config(fuse);
+		loop_config.clone_fd = opts.clone_fd;
+		loop_config.max_idle_threads = config->max_idle_threads;
+		res = fuse_loop_mt(fuse, &loop_config);
```

Comment:
This is ugly. `fuse` already includes the value for `max_idle_threads`, so we're effectively passing it twice. This is the result of treating `clone_fd` and `max_idle_threads` differently, and yet another reason to put them on the same footing :-).